src/components/InvitationModal/stylesheet.scss

```diff
@@ -0,0 +1,245 @@
+.invitation-modal {
+    margin: -20px;
+}
+
+.invitation-modal-content {
+    text-align: center;
+    color: white;
+}
+
+.email-block {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding-bottom: 5px;
+}
+
+// form {
+//     display: flex;
+//     flex-direction: column;
+// }
+
+//Style for email input
+
+.email-input {
+    display: flex;
+    flex-direction: column;
+}
+
+input {
+    width: 282px;
+    height: 24px;
+}
```

Comment:
All of the styles in this file need to be scoped to only apply to descendants of the `.invitation-modal` root element, since the stylesheet SCSS files in this project aren't isolated otherwise.

This might look like:

```scss
.invitation-modal {
  margin: -20px;
  // ... any other styles that apply to `.invitation-modal`

  input {
    width: 282px;
    height: 24px;
  }

  // ... all other selectors in this file
}
```

For a demonstration of the problem, see how this `input` selector/style block also applies to all other `<input>` elements in the application, such as the "course search" input box:

![Untitled-1](https://user-images.githubusercontent.com/26242455/220109449-d6137528-aa58-4916-bbaa-810fa6a10807.png)